runtime/runtime-params-estimator/src/wasmer_estimator.rs

```diff
@@ -98,3 +100,40 @@ pub fn cost_per_op(gas_metric: GasMetric) -> Ratio<u64> {
         NUM_ITERATIONS * outcome.burnt_gas,
     )
 }
+
+const RATIO_PRECISION: u64 = 1_000;
+
+fn compile(code: &[u8], gas_metric: GasMetric) -> f64 {
+    let prepared_code = prepare::prepare_contract(code, &VMConfig::default()).unwrap();
+    let start = start_count(gas_metric);
+    for _ in 0..NUM_ITERATIONS {
+        wasmer_runtime::compile(&prepared_code).unwrap();
```

Comment:
Can we instead add something like that into runner and select the VM using VMKind? This way dependency will go and we can compute costs for different runtimes.

Reply:
Good idea.